geoportailv3/static/less/layout.less

```diff
@@ -90,3 +90,70 @@ body {
     bottom: 0;
   }
 }
+
+.ol-control button {
+    width:41px;
+    height:41px;
+}
+
+.ol-control {
+    background-color: transparent;
+}
+
+.ol-control:hover {
+    background-color: transparent;
+}
+
+.ol-control button {
+    font-size: 1.5em;
+    font-weight:normal;
+    text-align: center;
+    text-indent: 0;
+    border: 1px solid #8394a0;
+    font-family: "geoportail-icons" !important;
+    margin: -1px;
+}
+
```

Comment:
What about taking advantage of less here?
The above could be simplified to something looking like the following:

```less
.ol-control {
  &, &:hover { // not tested
    background-color: transparent;
  }
  button {
    ...
  }
}
```